BUILDING.md

```diff
@@ -75,22 +86,26 @@ Node.js does not support a platform version if a vendor has expired support
-| GNU/Linux    | Tier 1       | kernel >= 3.10, glibc >= 2.17   | arm64            |                               |
-| macOS/OS X   | Tier 1       | >= 10.11                        | x64              |                               |
-| Windows      | Tier 1       | >= Windows 7/2008 R2/2012 R2    | x86, x64         | [1](#fn1),[2](#fn2),[3](#fn3) |
-| SmartOS      | Tier 2       | >= 16                           | x64              |                               |
-| FreeBSD      | Tier 2       | >= 11                           | x64              |                               |
-| GNU/Linux    | Tier 2       | kernel >= 3.13.0, glibc >= 2.19 | ppc64le >=power8 |                               |
-| AIX          | Tier 2       | >= 7.1 TL04                     | ppc64be >=power7 |                               |
-| GNU/Linux    | Tier 2       | kernel >= 3.10, glibc >= 2.17   | s390x            |                               |
-| GNU/Linux    | Experimental | kernel >= 2.6.32, glibc >= 2.12 | x86              | limited CI                    |
-| Linux (musl) | Experimental | musl >= 1.0                     | x64              |                               |
-
-<em id="fn1">1</em>: Tier 1 support for building on Windows is only on 64-bit
-  hosts. Support is experimental for 32-bit hosts.
+| Operating System | Architectures    | Versions                        | Support Type | Notes                             |
+| ---------------- | ---------------- | ------------------------------- | ------------ | --------------------------------- |
+| GNU/Linux        | x64              | kernel >= 3.10, glibc >= 2.17   | Tier 1       | e.g. Ubuntu 16.04, Debian 9, EL 7 |
+| GNU/Linux        | x64              | kernel >= 3.10, musl >= 1.1.19  | Tier 2       | e.g. Alpine 3.8                   |
+| GNU/Linux        | x86              | kernel >= 3.10, glibc >= 2.17   | Experimental |                                   |
+| GNU/Linux        | arm64            | kernel >= 4.5, glibc >= 2.17    | Tier 1       | e.g. Ubuntu 16.04, Debian 9, EL 7, [1](#fn1) |
+| GNU/Linux        | armv7            | kernel >= 4.14, glibc >= 2.24   | Tier 1       | e.g. Ubuntu 18.04, Debian 9       |
+| GNU/Linux        | armv6            | kernel >= 4.14, glibc >= 2.24   | Experimental | Downgraded as of Node.js 12       |
+| GNU/Linux        | ppc64le >=power8 | kernel >= 3.13.0, glibc >= 2.19 | Tier 2       | e.g. Ubuntu 16.04, EL 7           |
+| GNU/Linux        | s390x            | kernel >= 3.13.0, glibc >= 2.19 | Tier 2       | e.g. EL 7                         |
+| Windows          | x64, x86 (WoW64) | >= Windows 7/2008 R2/2012 R2    | Tier 1       | [2](#fn2),[3](#fn3)               |
+| Windows          | x86 (native)     | >= Windows 7/2008 R2/2012 R2    | Experimental |                                   |
+| Windows          | arm64            | >= Windows 10                   | Experimental |                                   |
```

Comment:
@joaocgreis please 👍 if this is correct

Reply:
We don't actually have any x86 Windows machines in CI. While running on WoW64 should give us plenty of confidence the binaries work, we can't claim Tier 1 level testing. What about leaving the `x86 (native)` line here as only experimental and clarifying it in the footnote? With something like:

```
<em id="fn6">6</em>: Running Node.js on x86 Windows should work and binaries
are provided. However, tests on our infrastructure only run on WoW64.
Furthermore, compiling on x86 Windows is currently considered Experimental and
may not be possible.
```